### ISSUE TITLE
Should a figure have a minimum size?

### ISSUE THREAD

**Issue by maartenbreddels** [bqplot/bqplot#1095]
Trying to fix #1085 I found the source of these pesky error msg'es:
![image](https://user-images.githubusercontent.com/1765949/80483484-761c9980-8956-11ea-8156-8ad6190ef60e.png)

The basically come from the figure having a dom size of 0, but having margins of 60 (on all sides). I wonder if we should give the plotarea_width (and height) and the dom element a minimum value of the sum of its margins.

**Comment by martinRenou** [comment]
:+1:

**Comment by SylvainCorlay** [comment]
OK, then it should it be computed based on the figure margins?

**Comment by maartenbreddels** [comment]
Yes, I think it should be such that the minimum figure size is `(0,0)`

**Comment by martinRenou** [comment]
Shouldn't we have a strictly positive minimum size? So that something gets rendered on the screen.
